packages/frontend/editor-ui/index.html

```diff
@@ -7,7 +7,6 @@
 		<link rel="icon" href="/favicon.ico" />
         <link rel="stylesheet" href="/static/prefers-color-scheme.css">
 		%CONFIG_SCRIPT%
-		<script src="/{{REST_ENDPOINT}}/sentry.js" type="text/javascript"></script>
```

Comment:
The `/sentry.js` endpoint has become part of `/config.js` and doesn't exist anymore.